rpc/config.go

```diff
@@ -30,21 +35,35 @@ type Config struct {
 
 // Web3RpcCmd creates a CLI command to start RPC server
 func Web3RpcCmd(cdc *codec.Codec) *cobra.Command {
-	return lcd.ServeCommand(cdc, registerRoutes)
+	cmd := lcd.ServeCommand(cdc, registerRoutes)
+	// Attach flag to cmd output to be handled in registerRoutes
+	cmd.Flags().String(flagUnlockKey, "", "Select a key to unlock on the RPC server")
+	return cmd
 }
 
 // registerRoutes creates a new server and registers the `/rpc` endpoint.
 // Rpc calls are enabled based on their associated module (eg. "eth").
 func registerRoutes(rs *lcd.RestServer) {
 	s := rpc.NewServer()
-	apis := GetRPCAPIs(rs.CliCtx)
+	accountName := viper.GetString(flagUnlockKey)
+
+	var emintKey emintcrypto.PrivKeySecp256k1
+	if len(accountName) > 0 {
```

Comment:
The command should fail if the account doesn't exist or can't be unlocked

Reply:
do you suggest a panic then? The base register routes functionality doesn't handle errors. Figured it would be sufficient to log the error since it is an optional field but I do agree.